public/video-ui/src/services/VideosApi.js

```diff
@@ -207,6 +231,9 @@ export default {
     return pandaReqwest({
       url: composerUrl,
       method: 'post',
+      headers: {
```

Comment:
Do we need this header for a Composer request?

Reply:
You are quite right, too much copy+paste